### PR TITLE
Splits psy points into tactical and strategic points

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request

Per title, turrets + phero towers use tactical points rn
All normal sources give 1x normal as strategic and .25x as tactical rn
## Why It's Good For The Game

See design doc, encourages not just spamming silos/towers

## Changelog
:cl:
add: Psy points have been split into tactical and strategic psy points
/:cl:

### CODE REVIEW

**Comment by Dikozda** [comment]
would be nice to have phero towers use the tactical points

**Comment by DeltaFire15** [comment]
Seems a bit scuffed to have xenos start with that is effectively 6 acid / 12 resin turrets with no drop in other things.
Then once the round reaches the point where you'd want to make additional structures, non-tactical ones become very difficult to acquire as cocoons commonly are only viable to defend if on a wipe.

**Comment by AlexCrimson** [comment]
Seems like a massive nerf. Silos/towers are required for Xenos to keep up with Marines. Cocooning is only viable if you wipe a group of Marines, which is a rare thing. Even then one dude with a drop pod can jump in and pop the cocoons to deny a crazy amount of psy.

Turrets themselves are shit most of the time. Turns out a static structure with limited range is not that great against enemies that can shoot across a screen. Mostly useful in caves where things are more narrow.

Given the limited amount of structures Xenos have, this just does not seem like its worth the added complexity.

**Comment by blackdav123** [comment]
this wont fix the issue of xenos just spamming silos and shit because they'll just cocoon everyone so they can still scale into the late game

it would make much more sense to split all psypoints earned into the two categories so xenos can still get their silos while using the other portion on tactical things

**Comment by FosterGrime-Heymynameslouie** [comment]
> > Drains give tactical only and cocoon strategic only for now
> 
> This will make things more confusing, especially for new players. I think that both just giving a portion as tactical would be better. Turrets can be very good, it's just that there's so many ways to cheese them I never want to place them down. Having points that can only be spent on things like turrets would definitely encourage using them.

whos going to play a xeno caste that uses psy/strat points as a new player? and plus, tac points is going to be used by all xenos that can build

I think the xeno doc rework is pretty cool https://hackmd.io/@uckgLKizQP6QNGj1w6mHQg/r14h9qS9h

**Comment by Wisemonster** [comment]
If psy drain and cocoon are going to give different psy points, then it would probably be a good idea to reflect this in their ability descriptions, and the changelog.

**Comment by Arturlang** [comment]
Pheremone turrets should use tactical points, but otherwise sounds good, encourages xenos to put down turrets on coccoons too.

**Comment by Wisemonster** [comment]
Just a heads up, cl still says only turrets use tac points, despite phero towers using em as well now.

**Comment by Wisemonster** [comment]
When buying blessings, it currently doesn't say which points are needed for each purchase. Should probably change that.

**Comment by ivanmixo** [comment]
Give beans more tacpoints on hijack pretty please

**Comment by Lumipharon** [comment]
Smelliest maintainer for not updating pr desc.

**Comment by TiviPlus** [comment]
> When buying blessings, it currently doesn't say which points are needed for each purchase. Should probably change that.

true

File: code/modules/mob/living/carbon/xenomorph/hive_upgrades.dm [hunk 1]
```diff
@@ -117,9 +121,10 @@ GLOBAL_LIST_INIT(tier_to_primo_upgrade, list(
 	SHOULD_CALL_PARENT(TRUE)
 	if((flags_upgrade & UPGRADE_FLAG_ONETIME) && times_bought)
 		return FALSE
-	if(SSpoints.xeno_points_by_hive[buyer.hivenumber] < psypoint_cost)
+	var/points_requirement = (flags_upgrade & UPGRADE_FLAG_USES_TACTICAL) ? SSpoints.xeno_tactical_points_by_hive[buyer.hivenumber] : SSpoints.xeno_strategic_points_by_hive[buyer.hivenumber]
+	if(points_requirement < psypoint_cost)
 		if(!silent)
-			to_chat(buyer, span_xenowarning("You need [psypoint_cost-SSpoints.xeno_points_by_hive[buyer.hivenumber]] more points to request this blessing!"))
+			to_chat(buyer, span_xenowarning("You need [points_requirement] more [(flags_upgrade & UPGRADE_FLAG_USES_TACTICAL) ? "tacrical" : "strategic"] points to request this blessing!"))
```
Comment (Arturlang) [inline code/modules/mob/living/carbon/xenomorph/hive_upgrades.dm:127]: tacrical

**Comment by PibeCantina** [comment]
You cant buy silos after the first one

**Comment by PibeCantina** [comment]
> You cant buy silos after the first one

Actually, after some testing, nothing that uses strategic points cant be built after the first one, even if it's destroyed

**Comment by TiviPlus** [comment]
chief read the fuckin pr you have a skill issue

**Comment by Arturlang** [comment]
Spawners should be tactical too or nobodys gonna use them, maybe make them cheaper too

**Comment by Lumipharon** [comment]
Surely people would have reported any other bugs. Right? :^)

**Review by Lumipharon — APPROVED** [review]

**Pull request merged**